RadarSDK/RadarPoint.m

```diff
@@ -0,0 +1,66 @@
+//
+//  RadarPoint.m
+//
+//  Copyright © 2020 Radar Labs, Inc. All rights reserved.
+//
+
+#import "RadarPoint.h"
+#import "RadarPoint+Internal.h"
+
+#import "RadarCollectionAdditions.h"
+
+@implementation RadarPoint
+
++ (NSArray<RadarPoint *> *)pointsFromObject:(id)object
+{
+    if (!object || ![object isKindOfClass:[NSArray class]]) {
+        return nil;
+    }
+
+    return [(NSArray *)object radar_mapObjectsUsingBlock:^RadarPoint *_Nullable(id _Nonnull pointObj) {
```

Comment:
I know this might be "cleaner" since it's using `map` and is therefore immutable, but the convention in our other models is to loop over the array -- https://github.com/radarlabs/radar-sdk-ios/blob/00b00af254bdee5d77d27b1ec7075809c0b3c25a/RadarSDK/RadarAddress.m#L21-L27

Reply:
bumping this - do we want to move towards `mapObjects` over `for`?

Reply:
oops. let me update